### PR TITLE
feat: dutch locales

### PR DESCRIPTION
**Pull request by vinckr**
## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
- [x] I have referenced an issue containing the design document if my change introduces a new feature.
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security vulnerability. 
      If this pull request addresses a security vulnerability, 
      I confirm that I got approval (please contact [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push the changes.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] I have added the necessary documentation within the code base (if appropriate).

### CODE REVIEW

File: src/locales/nl.json [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+    "identities.messages.1070008": "Code opnieuw verzenden",
+    "identities.messages.1070009": "Doorgaan",
+    "identities.messages.1070010": "Herstelcode",
+    "identities.messages.1070011": "Verificatiecode",
+    "identities.messages.1070012": "Registratiecode",
+    "identities.messages.1070013": "Aanmeldingscode",
+    "identities.messages.1080001": "Er is een e-mail met een verificatielink naar het opgegeven e-mailadres gestuurd. Als je geen e-mail hebt ontvangen, controleer dan de spelling van het e-mailadres en zorg ervoor dat je het e-mailadres hebt gebruikt waarmee je hebt geregistreerd.",
+    "identities.messages.1080002": "Je hebt succesvol je e-mailadres geverifieerd.",
+    "identities.messages.1080003": "Er is een e-mail met een verificatiecode naar het opgegeven e-mailadres gestuurd. Als je geen e-mail hebt ontvangen, controleer dan de spelling van het e-mailadres en zorg ervoor dat je het e-mailadres hebt gebruikt waarmee je hebt geregistreerd.",
+    "identities.messages.4000001": "{reden}",
```
Comment (Benehiko) [inline src/locales/nl.json:74]: placeholders should not be translated. See this example of german https://github.com/ory/elements/blob/main/src/locales/de.json#L74

```suggestion
    "identities.messages.4000001": "{reason}",
```

File: src/locales/nl.json [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+    "identities.messages.4000002": "Eigenschap {eigenschap} ontbreekt.",
+    "identities.messages.4000003": "lengte moet >= {min_lengte} zijn, maar is {werkelijke_lengte}",
```
Comment (Benehiko) [inline src/locales/nl.json:76]:
```suggestion
    "identities.messages.4000003": "lengte moet >= {min_length} zijn, maar is {actual_length}",
```
https://github.com/ory/elements/blob/main/src/locales/de.json#L76

File: src/locales/nl.json [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+    "identities.messages.4000004": "komt niet overeen met patroon \"{patroon}\"",
```
Comment (Benehiko) [inline src/locales/nl.json:77]:
```suggestion
    "identities.messages.4000004": "komt niet overeen met patroon \"{pattern}\"",
```

File: src/locales/nl.json [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+    "identities.messages.4000005": "Het wachtwoord kan niet worden gebruikt vanwege {reden}.",
```
Comment (Benehiko) [inline src/locales/nl.json:78]:
```suggestion
    "identities.messages.4000005": "Het wachtwoord kan niet worden gebruikt vanwege {reason}.",
```

File: src/locales/nl.json [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+    "identities.messages.4000008": "De verstrekte verificatiecode is ongeldig, probeer het opnieuw.",
+    "identities.messages.4000009": "Kon geen aanmeldingsidentificatoren vinden. Ben je ze vergeten in te stellen? Dit kan ook worden veroorzaakt door een servermisconfiguratie.",
+    "identities.messages.4000010": "Account nog niet actief. Ben je vergeten je e-mailadres te verifiëren?",
+    "identities.messages.4000011": "Je hebt geen TOTP-apparaat ingesteld.",
+    "identities.messages.4000012": "Deze herstelcode is al gebruikt.",
+    "identities.messages.4000013": "Je hebt geen WebAuthn-apparaat ingesteld.",
+    "identities.messages.4000014": "Je hebt geen herstelcodes ingesteld.",
+    "identities.messages.4000015": "Dit account bestaat niet of heeft geen beveiligingssleutel ingesteld.",
+    "identities.messages.4000016": "De herstelcode is niet geldig.",
+    "identities.messages.4000017": "lengte moet <= {max_lengte} zijn, maar is {werkelijke_lengte}",
```
Comment (Benehiko) [inline src/locales/nl.json:90]:
```suggestion
    "identities.messages.4000017": "lengte moet <= {max_length} zijn, maar is {actual_length}",
```

File: src/locales/nl.json [hunk 6]
```diff
@@ -0,0 +1,172 @@
...
+    "identities.messages.4000018": "moet >= {minimum} zijn, maar is {werkelijke}",
```
Comment (Benehiko) [inline src/locales/nl.json:91]:
```suggestion
    "identities.messages.4000018": "moet >= {minimum} zijn, maar is {actual}",
```

File: src/locales/nl.json [hunk 7]
```diff
@@ -0,0 +1,172 @@
...
+    "identities.messages.4000019": "moet > {minimum} zijn, maar is {werkelijke}",
```
Comment (Benehiko) [inline src/locales/nl.json:92]:
```suggestion
    "identities.messages.4000019": "moet > {minimum} zijn, maar is {actual}",
```

File: src/locales/nl.json [hunk 8]
```diff
@@ -0,0 +1,172 @@
...
+    "identities.messages.4000020": "moet <= {maximum} zijn, maar is {werkelijke}",
```
Comment (Benehiko) [inline src/locales/nl.json:93]:
```suggestion
    "identities.messages.4000020": "moet <= {maximum} zijn, maar is {actual}",
```

File: src/locales/nl.json [hunk 9]
```diff
@@ -0,0 +1,172 @@
...
+    "identities.messages.4000021": "moet < {maximum} zijn, maar is {werkelijke}",
```
Comment (Benehiko) [inline src/locales/nl.json:94]:
```suggestion
    "identities.messages.4000021": "moet < {maximum} zijn, maar is {actual}",
```

**Review by Benehiko — CHANGES_REQUESTED** [review]
The JSON is incorrect on line 131 and needs a comma. 
The variables in the text `{ variable }` should not be translated. they are used to inject values into from Kratos.

**Comment by Benehiko** [comment]
I've fixed it up and it should be alright

**Review by Benehiko — APPROVED** [review]

**Comment by vinckr** [comment]
Thanks Alano!

**Pull request merged**